admin-dev/themes/new-theme/js/pages/categories/index.js

```diff
@@ -43,6 +43,8 @@ import textToLinkRewriteCopier from "../../components/text-to-link-rewrite-copie
 import ChoiceTree from "../../components/form/choice-tree";
 import FormSubmitButton from "../../components/form-submit-button";
 import TaggableField from "../../components/taggable-field";
+import ShowcaseCard from "../../components/showcase-card/showcase-card";
+import ShowcaseCardCloseExtension from "../../components/showcase-card/extension/showcase-card-close-extension";
```

Comment:
This is probably outside the scope of this PR but do we really need ShowcaseCard and ShowcaseCardCloseExtension ? Isn't it part of the same feature/behavior 🤔 ?

Reply:
Haha, I'm sure I've asked you the same question like 6 months ago. :smile: